index.html

```diff
@@ -123,20 +133,26 @@ <h4 class="panel-title">
                         Save password in your browser
                     </label>
                   </div>
+                  <div class="checkbox">
+                    <label class="control-label" for="ssl">
+                      <input type="checkbox" id="ssl" ng-model="settings.ssl">
+                      Encryption. <strong>Strongly recommended!</strong> Need help? Check below.
+                    </label>
+                  </div>
                   <div class="checkbox" ng-show="settings.savepassword">
                     <label class="control-label" for="autoconnect">
-                        <input type="checkbox" id="autoconnect" ng-model="settings.autoconnect">
+                        <input type="checkbox" id="autoconnect" ng-model="settings.autoconnect"  ng-disabled="settings.useTotp">
                         Automatically connect
                     </label>
                   </div>
                   <div class="checkbox">
-                    <label class="control-label" for="ssl">
-                      <input type="checkbox" id="ssl" ng-model="settings.ssl">
-                      Encryption. <strong>Strongly recommended!</strong> Need help? Check below.
+                    <label class="control-label" for="useTotp">
+                      <input type="checkbox" id="useTotp" ng-model="settings.useTotp">
+                      Use Time-based One-Time Password <a href="https://blog.weechat.org/post/2019/01/14/Support-of-TOTP">ℹ️</a>
```

Comment:
Link should be `target="_blank"` I guess, but this applies to the other links as well, so we should just change it everywhere at some point

Reply:
Also maybe more prominent. Bootstrap comes with some icons, maybe use those:
```html
<i class="glyphicon glyphicon-info-sign"></i>
```